WowPacketParserModule.V8_0_1_27101/Parsers/BattlePetHandler.cs

```diff
@@ -0,0 +1,108 @@
+﻿using WowPacketParser.Enums;
+using WowPacketParser.Misc;
+using WowPacketParser.Parsing;
+
+namespace WowPacketParserModule.V8_0_1_27101.Parsers
+{
+    public static class BattlePetHandler
+    {
+        private static readonly int[] _petBattleEffectTargets = { 0, 1, 2, 3, 4, 5, 6, 7 };
+
+        public static void ReadPetBattleEffectTarget(Packet packet, int[] targetTypeValueMap, params object[] idx)
+        {
+            var type = packet.ReadBits("Type", 4, idx); // enum PetBattleEffectTargetEx
+
+            packet.ResetBitReader();
+
+            packet.ReadByte("Petx", idx);
+
+            switch (targetTypeValueMap[type])
```

Comment:
uhh, this is exactly the same as `switch (type)`

Reply:
Gonna fix that